code/_onclick/item_attack.dm

```diff
@@ -8,6 +8,10 @@
  * * [/obj/item/proc/afterattack]. The return value does not matter.
  */
 /obj/item/proc/melee_attack_chain(mob/user, atom/target, params)
+	if (are_fingers_too_big(user, target))
+		balloon_alert(user, "fingers too big!")
+		return FALSE
```

Comment:
This early in the chain means you literally can't even place the offending item onto a table to get rid of it. Not a fan of this tbh

Reply:
Not to mention it's kinda snowflake, if we do go this direction i'd prefer if it was a blanket `can_melee_attack_chain_with`

Reply:
it also has a fatal flaw because i forgot about something
it was oranges' suggestion to put this into the attack chain and it sounded like a good idea at the time but it turns out (and this might suprise you) that the attack chain is quite complicated

i might need to get signalpilled for this